dotnet/src/dotnetframework/Providers/Storage/GXAzureStorage/AzureStorageExternalProvider.cs

```diff
@@ -387,13 +387,13 @@ public bool TryGetObjectNameFromURL(string url, out string objectName)
 			string baseUrl = StorageUri + StorageUtils.DELIMITER + PublicContainer.Name + StorageUtils.DELIMITER;
 			if (url.StartsWith(baseUrl))
 			{
-				objectName = url.Replace(baseUrl, string.Empty);
+				objectName = StorageUtils.StripQueryString(url.Replace(baseUrl, string.Empty));
```

Comment:
I think this code should not be necessary, as it is in: [StorageFactory.cs]. Or I missing something. 
(https://github.com/genexuslabs/DotNetClasses/blob/923666b14cf531ca9a5541f8c6c59457e6c9e806/dotnet/src/dotnetframework/GxClasses/Storage/StorageFactory.cs).

```
public static bool TryGetProviderObjectName(ExternalProvider provider, string objectNameOrUrl, out string providerObjectName)
		{
			providerObjectName = null;
			if (provider != null && provider.TryGetObjectNameFromURL(objectNameOrUrl, out providerObjectName))
			{
				int idx = providerObjectName.IndexOf(QUESTION_MARK);
				if (idx > 0)
				{
					providerObjectName = providerObjectName.Substring(0, idx);
				}
				return true;
			}
			return false;
		}
```

Reply:
Certainly. Replaced by StorageFactory.TryGetProviderObjectName.